chainerx_cc/chainerx/routines/creation.cc

```diff
@@ -345,4 +347,119 @@ Array Linspace(
     return out;
 }
 
+namespace {
+
+// Defines the backward pass for Meshgrid.
+void MeshgridBackward(const std::vector<Array>& inputs, const std::vector<Array>& outputs, const std::string& kind) {
```

Comment:
How about `MeshgridGrad` which itself has a routine-like interface and behavior. We have other routines which are implemented that way. Then, the graph would be created in `Meshgrid`, but inside the backward definition, `MeshgridGrad` would be called to do the computation of the gradient. What do you think?

Reply:
Turns out, that we can rely on existing machinery so there is no need for a `BackwardBuilder`.